code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -686,12 +758,13 @@
 	custom_metabolism = 0.2
 	overdose = REAGENTS_OVERDOSE/5
 	overdose_critical = REAGENTS_OVERDOSE_CRITICAL/5
+	scannable = 1
 
 	on_mob_life(mob/living/M)
 		. = ..()
 		if(!.) return
 
-		M.reagent_move_delay_modifier -= 0.5
+		M.reagent_move_delay_modifier -= 9
```

Comment:
What's the advantage of ultrazine over hyperzine in this context?

Reply:
Ultrazine provides a better speed bonus, lasts far longer, has a much higher OD ceiling, and no heart damage side effects. Hyperzine but better with no side effects except an addiction that is long deferred.